index.ts

```diff
@@ -39,7 +39,12 @@ async function getParser(
   parserFileName: string
 ): Promise<(arg0: Response) => void> {
   const parser = await import(`${PARSER_PATH}/${parserFileName}`);
-  return parser.default;
+  if (parser.default) {
+    return parser.default;
+  } else {
```

Comment:
nit: you don't need an "else" statement here, since regardless of the conditions you will execute this code anyway.
If you return `parser.default`, the code that follows will never run. If you don't return at that point, all the code that follows will run. 

Also, it's customary in most cases to ["Return Early"](https://knowthecode.io/code-tip-return-early). Basically, you want to run as less checks as possible before you execute logic that will not end up being returned, and also get validations out of the way as early as possible. So you could just do:
```typescript
if (!parser.default) {
    console.warn('The imported module must have an export default statement');
    process.exit(0);
}
return parser.default;
```